west.yml

```diff
@@ -0,0 +1,12 @@
+manifest:
+
+  projects:
+    - name: zephyr
+      url: https://github.com/zephyrproject-rtos/zephyr
+      revision: main
+      path: zephyr
+      west-commands: scripts/west-commands.yml
+      import:
+        name-allowlist:
+          - cmsis_6
```

Comment:
It might be a good idea to add `cmsis` along with `cmsis_6` because `cmsis_6` will be replaced by `cmsis`. Or do we permanently have the version suffix? I can't find the related issue in Zephyr repo.
https://docs.zephyrproject.org/latest/hardware/arch/arm_cortex_m.html#cmsis

Reply:
Okay, added `cmsis` along with `cmsis_6`.